hb_toolkit.py

```diff
@@ -118,30 +117,51 @@ def determineInviteeEligibility(self, invitee):
         """
         Takes a tuple of user_name, user_id, userpage_id.
         """
-        is_eligible = False
-        has_skip_template = False
-        is_blocked = self.getBlockStatus(invitee[1])
-        if invitee[2] is not None:
-            has_skip_template = self.checkTalkPage(self.output_params["output namespace"] + invitee[0], invitee[2], self.output_params["skip templates"])
+
+        is_blocked = self.getBlockStatus(invitee["id"])
+        if is_blocked:
+            print "User '{}' was not eligible; blocked.".format(invitee["name"])
+            return False
+        if invitee["talkpage_id"] is not None:
+            talkpage_path = "User talk:{}".format(invitee["name"])
+            has_skip_string = self.checkTalkPage(
+                talkpage_path,
+                invitee["talkpage_id"],
+                hb_config.skip_strings
+            )
+            if has_skip_string:
+                print "User '{}' was not eligible; has skip string.".format(invitee["name"])
+                return False
+            has_skip_template = self.checkTalkPage(
+                talkpage_path,
+                invitee["talkpage_id"],
+                hb_config.skip_templates,
+                "{{{{\s*{}\s*[}}|]"
```

Comment:
OK, I'll leave it then. Also, if you really want to use it, you can still add another string like "template:hej".